ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -728,6 +728,21 @@ MAU_DFSS_DFVSC_DSP_CHWDC_HWDC_OADM_ZSPM:
   opt_uses:
   - smoke_alarm
 
+MAU_DFSS_DFVSC_DSP_HWDDC_CHWDDC_OADM:
+  guid: "95e9e8ec-92a1-4cd9-b940-03370ea8728d"
+  description: "Makeup air unit with dual-setpoint heating and cooling water valve control on discharge side."
+  is_canonical: true
+  implements:
+  - MAU
+  - DFSS
+  - DFVSC
+  - DSP
```

Comment:
How can the unit have dual setpoints for the discharge and zone temps? Would expect one or the other